ui/app/src/components/shared/utils.ts

```diff
@@ -27,11 +27,11 @@ export function formatNumber(displayNumber: string) {
 
 // TODO: These could be replaced with a look up table
 export function getPeggedSymbol(symbol: string) {
-  if (symbol === "erowan") return "rowan";
-  return "c" + symbol;
+  if (symbol.toLowerCase() === "erowan") return "ROWAN";
+  return "c" + symbol.toUpperCase();
 }
 export function getUnpeggedSymbol(symbol: string) {
-  if (symbol === "rowan") return "erowan";
+  if (symbol.toLowerCase() === "rowan") return "eROWAN";
   return symbol.indexOf("c") === 0 ? symbol.slice(1) : symbol;
 }
```

Comment:
hmm... looks like there was an attempt to have these centralized but is was not implemented. Are these actually used anywhere?

Reply:
I believe no